packages/edit-site/src/components/sidebar-navigation-screen-details-footer/index.js

```diff
@@ -44,33 +43,32 @@ export default function SidebarNavigationScreenDetailsFooter( {
 		hrefProps.as = 'a';
 	}
 	return (
-		<ItemGroup className="edit-site-sidebar-navigation-screen-details-footer">
-			<SidebarNavigationItem
-				aria-label={ __( 'Revisions' ) }
-				{ ...hrefProps }
-				{ ...otherProps }
-			>
-				<SidebarNavigationScreenDetailsPanelRow justify="space-between">
+		<ItemGroup
+			size="large"
+			className="edit-site-sidebar-navigation-screen-details-footer"
+		>
+			<Item>
+				<SidebarNavigationScreenDetailsPanelRow>
 					<SidebarNavigationScreenDetailsPanelLabel>
-						{ __( 'Last modified' ) }
-					</SidebarNavigationScreenDetailsPanelLabel>
-					<SidebarNavigationScreenDetailsPanelValue>
 						{ createInterpolateElement(
 							sprintf(
 								/* translators: %s: is the relative time when the post was last modified. */
-								__( '<time>%s</time>' ),
+								__( 'Last modified: <time>%s</time>' ),
```

Comment:
Do you think it makes sense to use the [simpler format used by posts and templates](https://github.com/WordPress/gutenberg/blob/564f0699e0924b7114af55d51f1909de07a2e3c9/packages/editor/src/components/post-last-edited-panel/index.js#L20-L20)?


<img width="245" alt="Screenshot 2024-11-20 at 10 04 22 am" src="https://github.com/user-attachments/assets/687b6216-871f-4849-8a3f-06b3d30bc323">

```js
sprintf(
	// translators: %s: Human-readable time difference, e.g. "2 days ago".
	__( 'Last edited %s.' ),
	humanTimeDiff( modified )
);
```

Reply:
Actually, on second thought. How valuable is this information?

If it's still accessible, I think just having the revisions button with a count might appear less cluttered:

```js
// translators: %s: number of revisions.
sprintf( __( '(%s) Revisions' ), revisionsCount )
```

![Screenshot 2024-11-20 at 10 19 44 am](https://github.com/user-attachments/assets/81dff5c2-5fec-4889-85d0-30c0771471c2)

What do you reckon?